docs/data/data-grid/clipboard/clipboard.md

```diff
@@ -17,6 +17,16 @@ The priority of the data copied to the clipboard is the following, from highest
 
 ## Clipboard paste [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
 
+:::info
+
+<details style="margin: 0">
+  <summary markdown="span">Video demo</summary>
+  <video muted loop playsinline controls style="margin-top: 8px">
+    <source src="/static/x/clipboard-paste.mov" type="video/mp4" />
```

Comment:
Just wondering if we could load it from github CDN to keep docs build smaller and save some bandwidth on our side 😄 
```suggestion
    <source src="https://github-production-user-asset-6210df.s3.amazonaws.com/13808724/237996024-abfcb5c6-9db6-4677-9ba7-ae97de441080.mov" type="video/mp4" />
```

Reply:
Initially, I added the video to our docs to have full control over it. But I think using the one hosted by GitHub should work just fine and we use it in our changelog anyway 👍